R/learner_xgboost_surv_xgboost.R

```diff
@@ -6,24 +6,22 @@
 #' eXtreme Gradient Boosting regression.
 #' Calls [xgboost::xgb.train()] from package \CRANpkg{xgboost}.
 #'
+#' **Note:** We strongly advise to use the separate [Cox][LearnerSurvXgboostCox]
```

Comment:
maybe you want to add a deprecated warning to the old xgboost learner?

Reply:
then at some point later we can remove it as you suggested.

Reply:
Added one yes